db/migrate/20231020091031_convert_unit_amount_cents_to_decimal.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ConvertUnitAmountCentsToDecimal < ActiveRecord::Migration[7.0]
+  def change
+    change_column :fees, :unit_amount_cents, :decimal, precision: 30, scale: 5, null: false
```

Comment:
Rails/ReversibleMigration: change_column is not reversible. (https://rails.rubystyle.guide#reversible-migration, https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html)